astro.config.mjs

```diff
@@ -3,6 +3,8 @@ import mdx from '@astrojs/mdx';
 import sitemap from '@astrojs/sitemap';
 import remarkBreaks from 'remark-breaks'; // improves support for newlines in markdown files
 import remarkGfm from 'remark-gfm'; // support rendering tables in markdown files
+import remarkMath from 'remark-math';
+import rehypeKatex from 'rehype-katex';
```

Comment:
This is a fix for a warning that occurs in the command line tools as soon as `mdx()` plugin below is enabled.